set.mm

```diff
@@ -716530,6 +716542,1358 @@ contain the empty set ( ` -. (/) e. S ` ), no additional assumption is
 $( End $[ set-mbox-av-afv_aov.mm $] $)
 
 
+$( Begin $[ set-mbox-av-n0set.mm $] $)
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  The restricted universal class - class of inhabited sets
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
+
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+  General auxiliary theorems
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+$)
+
+  ${
+    nelcon3i.1 $e |- ( A e. B -> C e. D ) $.
+    $( Contrapositive law deduction for negated membership, inference form.
+       (Contributed by AV, 1-Apr-2022.) $)
+    nelcon3i $p |- ( C e/ D -> A e/ B ) $=
+      ( wcel wn wnel con3i df-nel 3imtr4i ) CDFZGABFZGCDHABHMLEICDJABJK $.
+  $}
+
+  ${
+    $d A x $.  $d B x $.  $d C x $.
+    $( Two superclasses are equal if and only if they are equal without a
+       common subset.  (Contributed by AV, 30-Mar-2022.) $)
+    rcompleq1 $p |- ( ( C C_ A /\ C C_ B )
+                      -> ( A = B <-> ( A \ C ) = ( B \ C ) ) ) $=
+      ( vx wss wa wceq cdif wcel wi ssel com12 eldif eldifi sylbir expcom com23
+      adantld impd pm2.61i difeq1 cv ad3antlr wn eleq2 syl6bi ad3antrrr syl6bir
+      impbida eqrdv ex impbid2 ) CAEZCBEZFZABGZACHZBCHZGZABCUAUOUSUPUOUSFZDABUT
+      DUBZAIZVABIZVACIZUTVBFZVCJVEVDVCUNVDVCJUMUSVBCBVAKUCLVDUDZUTVBVCVFVBUTVCV
+      BVFUTVCJZVBVFFVAUQIZVGVAACMVHUSVCUOUSVHVCUSVHVAURIZVCUQURVAUEZVABCNUFLROP
+      QSTVDUTVCFZVBJVKVDVBUMVDVBJUNUSVCCAVAKUGLVFUTVCVBVFVCUTVBVCVFUTVBJZVCVFFV
+      IVLVABCMVIUSVBUOUSVIVBUSVIVHVBVJVAACNUHLROPQSTUIUJUKUL $.
+  $}
+
+  $( The difference between a class and the universe is empty.  (Contributed by
+     AV, 30-Mar-2022.) $)
+  difv0 $p |- ( A \ _V ) = (/) $=
+    ( cvv wss cdif c0 wceq ssv ssdif0 mpbi ) ABCABDEFAGABHI $.
```

Comment:
This looks like it's general and interesting enough to be moved to the main part  -  let's see.